eos/models/wet_TEST.cc

```diff
@@ -283,7 +283,7 @@ class WilsonCoefficientsSBNuNuTest :
             {
                 static const double eps = 1e-8;
 
-                Parameters p = reference_parameters();  
+                Parameters p = Parameters::Defaults();  
```

Comment:
Can you remove `reference_parameters` entirely and trade it for `Parameters::Defaults` wherever it is used in this file?